### DISCUSSION TITLE
Plugin's library not opening file picker

### DISCUSSION THREAD

**Discussion by Spectre-856** [tsukasa/BdBrowser#33] [Problems and issues]
When I click on the plugin's folder in the settings, this appears, idk what to do

![Captura de tela 2024-03-14 191851](https://github.com/tsukasa/BdBrowser/assets/163223464/f39ce5c9-3ebd-4cd9-867d-947c087b2e6b)

**Comment by tsukasa** [comment]
Greetings Spectre-856,

thank you for the report! I have verified and reproduced the issue. It is due to changes in how BetterDiscord handles opening paths in its latest version. Previously BD called electron.shell.openItem(), whereas now it sends the IPC command bd-open-path you observed in your screenshot. Since BdBrowser does not yet support handling bd-open-path properly, I need to refactor some code and run some tests to ensure no new regressions crop up.

You should expect a fix for this issue sometime this weekend. Apologies for the inconvenience!

**Comment by tsukasa** [comment]
[The release 1.10.0.20240315](https://github.com/tsukasa/BdBrowser/releases/tag/v.1.10.0.20240315) contains the necessary adjustments to restore the intended behaviour. Thank you for your patience and for taking the time to file bug reports!

**Reply by Spectre-856** [reply]
Thanks dude, appreciate your work, Im really grateful
